app/models/patient.rb

```diff
@@ -19,34 +19,18 @@ class Patient < ApplicationRecord
     end
   end
 
-  validates :monitoring_reason, inclusion: { in: ['Completed Monitoring',
-                                                  'Enrolled more than 14 days after last date of exposure (system)',
-                                                  'Enrolled more than 10 days after last date of exposure (system)',
-                                                  'Enrolled on last day of monitoring period (system)',
-                                                  'Completed Monitoring (system)',
-                                                  'Meets Case Definition',
-                                                  'Lost to follow-up during monitoring period',
-                                                  'Lost to follow-up (contact never established)',
-                                                  'Transferred to another jurisdiction',
-                                                  'Person Under Investigation (PUI)',
-                                                  'Case confirmed',
-                                                  'Past monitoring period',
-                                                  'Meets criteria to discontinue isolation',
-                                                  'Deceased',
-                                                  'Duplicate',
-                                                  'Other',
-                                                  nil, ''] }
-
+  validates :monitoring_reason, inclusion: {
+    in: VALID_ENUMS[:monitoring_reason],
```

Comment:
these are included as options here in the valid enums: https://github.com/SaraAlert/SaraAlert/blob/SARAALERT-950/app/helpers/validation_helper.rb#L87

am I missing anything?

Reply:
I don't think you are, I do see nil and '' in that list. However, `VALID_ENUMS` might not be a specific enough name, since it's defined in `app/helpers/validation_helper.rb#L87`, which I suppose could be used for validations for any model. Could we call it something like `VALID_PATIENT_ENUMS`?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-4f61e388997f8d92e41f66947a47b461707b615cf9fac69f1080b8254630d2ceR75

Reply:
Yep thanks Tim, missed those on my first look through.